.github/workflows/ci.yml

```
@@ -12,11 +12,11 @@ jobs:
       fail-fast: false
       matrix:
         go-version:
-          - 1.15
-          - 1.16
           - 1.17
           - 1.18
           - 1.19
+          - 1.20
+          - 1.21
```

```suggestion
          - '1.17'
          - '1.18'
          - '1.19'
          - '1.20'
          - '1.21'
```
to avoid being parsed as number

---

(`1.20` is parsed as `1.2`)

---

Ah, thanks. Fixed.